src/mame/heathkit/z37_fdc.cpp

```diff
@@ -49,21 +49,21 @@ void heath_z37_fdc_device::ctrl_w(u8 val)
 {
 	bool motor_on = bool(BIT(val, ctrl_MotorsOn_c));
 
-	m_intrq_allowed = bool(BIT(val, ctrl_EnableIntReq_c));
+	m_irq_allowed = bool(BIT(val, ctrl_EnableIntReq_c));
 	m_drq_allowed = bool(BIT(val, ctrl_EnableDrqInt_c));
-	m_fdc->dden_w(BIT(val, ctrl_SetMFMRecording_c) ? CLEAR_LINE : ASSERT_LINE);
+	m_fdc->dden_w(!BIT(val, ctrl_SetMFMRecording_c));
```

Comment:
In other places you seem to be avoiding implicit Boolean-to-integer conversion, but you’ve added one here.  Should it be `m_fdc->dden_w(BIT(~val, ctrl_SetMFMRecording_c));` for consistency with the pattern of avoiding the implicit conversions?